docs/janssen-server/usermgmt/usermgmt-scim.md

```
@@ -75,11 +75,11 @@ You can refer to [here](../../janssen-server/config-guide/scim-config/user-confi
 
 You can refer to [here](../../janssen-server/config-guide/scim-config/user-config.md#get-access-token) for this topic
 
-### Retrive existing User 
+### Retrieve existing User
 
-You can refer to [here](../../janssen-server/config-guide/scim-config/user-config.md#retrive-existing-user) for this topic
+You can refer to [here](../../janssen-server/config-guide/scim-config/user-config.md#retrieve-existing-user) for this topic
 
-## Creating Resource 
+## Creating Resource
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)